src/transformer/events/mod_quiz/question_answered/multichoice.php

```diff
@@ -45,11 +49,16 @@ function multichoice(array $config, \stdClass $event, \stdClass $questionattempt
                     $lang => $question->questiontext,
                 ],
                 'interactionType' => 'choice',
+                "correctResponsePatter" => [$questionattempt->rightanswer],
+            ],
+            'extensions' => [
+                'answers' => $answerarray
```

Comment:
Firstly, in the test this is in the result extensions not the object extensions. Secondly, the key needs to be a URL. Thirdly, this probably won't be merged since it'll be too much data for some of the installers with larger volumes of users.